src/components/views/ViewDataTable/index.tsx

```diff
@@ -53,10 +54,20 @@ const ViewDataTable: FunctionComponent<ViewDataTableProps> = ({ columns, rows, v
     const [selection, setSelection] = useState<number[]>([]);
     const [error, setError] = useState<VIEW_DATA_TABLE_ERROR>();
     const [waiting, setWaiting] = useState(false);
+    const [sortModel, setSortModel] = useState<GridSortModel>([]);
     const router = useRouter();
     const { orgId } = router.query;
     const queryClient = useQueryClient();
     const viewId = view.id.toString();
+    const { showSnackbar } = useContext(SnackbarContext);
+
+    /* eslint-disable react-hooks/exhaustive-deps */
+    useEffect(() => {
+        if (error) {
+            showSnackbar('error', intl.formatMessage({ id: `misc.views.dataTableErrors.${error}` }) );
+            setError(undefined);
+        }
+    }, [error]);
```

Comment:
I think instead of setting errors this way, we could just call `showSnackbar` any time we would use `setError`.